### ISSUE TITLE
Anyone got this working with 6 digits?

### ISSUE THREAD

**Issue by wouterds** [bremme/arduino-tm1637#51]
[![image](https://user-images.githubusercontent.com/1210628/109383972-2c8c3a80-78ea-11eb-9515-9ad76c2b6fb6.png)](https://www.aliexpress.com/item/1005001530959875.html)

**Comment by ABX00032** [comment]
Yes you need Specific Library called - TM1637TinyDisplay6.h by Author: Jason A. Cox - @jasonacox - https://github.com/jasonacox.
